### PR TITLE
refactor!: make the package follow appium2 modules

### PR DESCRIPTION
**Pull request by KazuCocoa**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const gulp = require('gulp');
-const boilerplate = require('appium-gulp-plugins').boilerplate.use(gulp);
+const boilerplate = require('@appium/gulp-plugins').boilerplate.use(gulp);
```
Comment (mykola-mokhnach) [inline gulpfile.js:4]: The package won't work after publish if we don't add `index.js` explicitly to boilerplate files. See for example https://github.com/appium/appium-uiautomator2-server/pull/455/files#diff-25789e3ba4c2adf4a68996260eb693a441b4a834c38b76167a120f0b51b969f7R14

Comment (KazuCocoa) [reply gulpfile.js:4]: I see. As https://github.com/appium/node-simctl/pull/153/commits/7bf2efa5c4f3a444e88063acbf153576021690c6 ?
This repository does not have a top-level index.js, so this repo needs it.

Comment (mykola-mokhnach) [reply gulpfile.js:4]: you are right, this is not needed for this repository: "main": "./build/lib/simctl.js",

File: index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import Simctl from './lib/simctl';
```
Comment (mykola-mokhnach) [inline index.js:1]: if you add it then make sure package.js is pointing to it, but I'd say lets keep everything as it was

**Comment by mykola-mokhnach** [comment]
Please test the newly added package changes carefully after publishing it. Otherwise LGTM

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**